src/components/People/PeopleTable.tsx

```diff
@@ -0,0 +1,121 @@
+import cn from 'classnames';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isTableLoading, setIsTableLoading] = useState(false);
+  const [apiError, setApiError] = useState('');
+
+  const { personSlug } = useParams();
+
+  const findMother = (motherName: string | null) => {
+    const mother = people?.find(person => person.name === motherName);
+
+    return mother || null;
+  };
+
+  const findFather = (fatherName: string | null) => {
+    const father = people?.find(person => person.name === fatherName);
+
+    return father || null;
+  };
+
+  useEffect(() => {
+    setIsTableLoading(true);
+    getPeople()
+      .then((peopleFromServer) => {
+        setPeople(peopleFromServer);
+      })
+      .catch(() => {
+        setApiError('unable to load data');
```

Comment:
Since you are not using this message, it is better to replace it by a boolean  such as (setHasError(true))